### ISSUE TITLE
Ags Never launches nor Creates a Widget, Always spits out error regardless of what I have

### ISSUE THREAD

**Issue by MilkJug1** [Aylur/ags#226]
I am a new user to ags, I might be doing something wrong but I have no  idea.

# Issue
 Ags, regardless of config editing, will always be spitting out this error that is below this text, and never launch any widget. I have followed the tutorial to a tee and couldn't seem to get this working, even when using an example.

I have ran ags with the follow command. `ags --config $HOME/.config/ags/config.js` and the same issues still arise.

```
(ags:12214): Gjs-WARNING **: 01:08:05.399: JS ERROR: SyntaxError: import declarations may only appear at top level of a module @ /home/milk/.config/ags/config.js:1:0

(ags:12214): Gjs-WARNING **: 01:08:05.399: JS ERROR: SyntaxError: import declarations may only appear at top level of a module @ /home/milk/.config/ags/config.js:1:0
Gjs-Message: 01:08:05.399: JS LOG: ERROR: No config was provided or there was an error

(ags:12214): Gjs-WARNING **: 01:08:05.399: JS ERROR: SyntaxError: import declarations may only appear at top level of a module @ /home/milk/.config/ags/config.js:1:0
```
![image](https://github.com/Aylur/ags/assets/76970674/613a8bf6-a99a-4696-9fe1-fbbdd2ceffc3)
A image showing my terminal output, with all of what my config.js file has. 

Now this was when my `~/.config/ags/config.js` had looked the very first javascript block from the Configuration part of the wiki, I thought this was due to the fact nothing had been defined in the `windows` section. So I decided to copy the code from one of the examples, the Simple Bar example to be exact.

![image](https://github.com/Aylur/ags/assets/76970674/c64df1d3-860c-463d-9aa8-14fa12625c42)
> [!NOTE]
> I had already relaunched ags when I had copied the code from the example files in the photo, same errors, but no widget to pop up. 

When using the Simple Bar Example files and trying to use `ags toggle-window Bar`, in order to get the bar to appear, ags spits out an error saying: 
```
Gjs-Message: 01:18:41.341: JS LOG: WARNING: There is no window named Bar
```
![image](https://github.com/Aylur/ags/assets/76970674/64618126-c978-436a-915a-f68424b42087)

Now I have no idea what I am doing wrong, or if this is something to do with Javascript, I have reinstalled ags multiple times.
For my system info, here it is:

- Distro: Arch Linux x86_64
-  Window Manager: Hyprland
- Linux Kernel Version: 6.6.7-arch1-1
- ags Version: 1.5.5-1

I installed ags from the AUR with paru, now I'm going to manually compile from scratch and see if that fixes. I'm sorry if this issue is bad, I don't really know how to explain this.

I have tried to provide as much info as I can, I see this issue is similar to #55, but it was closed without the person giving the solution.

**Comment by MilkJug1** [comment]
I can and will provide any more info, if needed, just let me know. Hopefully I gave as much as I was able to.

**Comment by Aylur** [comment]
Weird, what is your gjs version?
```js
gjs --version
```

**Comment by MilkJug1** [comment]
> Weird, what is your gjs version?
> 
> ```js
> gjs --version
> ```

`gls 1.78.1`
When running `gjs --version`

**Comment by MilkJug1** [comment]
Um, sorry to ask, but why was this closed? I still have been having trouble getting this to work. Though, I will try again tomorrow to get it to work.

**Comment by MilkJug1** [comment]
Okay I think I may have found the issue on why it wasn't working before. For some reason on my system, there is a really old version of ags, I have no idea how it got installed or which package is using it (I even uninstalled everything related to ags that I could find). Looking through the PKGBUILD file from the AUR, I went to see if running the exact path of the excutable would fix it. Now I got a different output, though I am having another issue. It's seperate from Ags as I was dealing with this issue on waybar, but I think I may have figured out why Ags wasn't working before. I, for some weird reason, had a version 0.1.0 of ags.


![image](https://github.com/Aylur/ags/assets/76970674/086c94fa-f987-4531-9343-101e47183f46)
